New version: AffineArithmetic v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: AffineArithmetic
- Repository: https://github.com/JuliaIntervals/AffineArithmetic.jl
- Created by: @OlivierHnt
- Version: v0.3.0
- Commit: d1643d9575b252f09c256569754b415c2524c192
- Reviewed by: @OlivierHnt
- Reference: https://github.com/JuliaIntervals/AffineArithmetic.jl/commit/d1643d9575b252f09c256569754b415c2524c192#commitcomment-165235688
- Description: Affine arithmetic in Julia
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- Compatibility with IntervalArithmetic 0.22
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6f350d7f3d63fe09ac3fd880d5799c7ced97d9e2205d65c121701f66f90d1071b77f539a0026ead63083482de703294a58e5ba7a159dc02fe8bb06f4d1485cc60089f7344ca47e5533cf5eae95657b1cf47b37912f7608f1b463b02dda17c3261edbb630aaeec47b9f4f4e946f6c7e7b8bb30e5761563f24a8b16e6b925cbbd76554cdfc1cd7342f1b94dd2abf8bc08fcda3efca50a2226e1f4da2e74c69921056fca6b80ad2e06490291897e806e9febe000f7c860c55a9f002156c8c5c500d7 -->

OlivierHnt commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- Compatibility with IntervalArithmetic 0.22

JuliaRegistrator committed:
New version: AffineArithmetic v0.3.0

UUID: 2e89c364-fad6-56cb-99bd-ebadcd2cf8d2
Repo: https://github.com/JuliaIntervals/AffineArithmetic.jl.git
Tree: 3431b7e134840519ee3135ef72348848e41e239e

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

OlivierHnt commented:
[noblock]
